core/components/atoms/breadcrumbs/__stories__/LabelTruncate.story.jsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Breadcrumbs, PageHeader } from '@/index';
+import { action } from '@/utils/action';
+
+// CSF format story
+export const labelTruncate = () => {
+  const breadcrumbs = (
+    <Breadcrumbs
+      list={[
+        {
+          label: 'eCQM Performance year 2022',
+          link: '/eCQM',
+        },
+        {
+          label: 'Report 1',
+          link: '/report1',
+        },
+      ]}
```

Comment:
Please create a story for more than 4 level also depicting label overflow behaviour.